src/components/helpers/get-age-string.test.js

```diff
@@ -0,0 +1,28 @@
+import getAge from "./get-age-string";
```

Comment:
I see, that when running these tests locally, it says some Deprecation warning message about usage of moment package.
I know, that it's not related to your test changes, but it related to file, which you try to test. Please fix this warning and I will approve

Reply:
fixed.